src/patterns/components/icons/react/global-info.hbs

```diff
@@ -0,0 +1,39 @@
+<p class="sprk-u-mbm">The sprk-icon component expects that the icon set has been
+  imported. See below for available customization options:</p>
+<table class="drizzle-c-Table drizzle-c-Table--wide sprk-u-mbm">
+  <colgroup>
+    <col class="sprk-u-Width-40">
+    <col>
+  </colgroup>
+  <thead>
+  <tr>
+    <th>Component</th>
+    <th>Prop</th>
+    <th>Type</th>
+    <th>Description</th>
+  </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td>SprkIcon</td>
+    <td class="sprk-u-FontWeight--bold">iconType</td>
```

Comment:
What do we think about using iconName instead of iconType? The docs for this say that it expects the name of the icon. It might be more clear to use name.